lib/puppet/provider/service/upstart.rb

```diff
@@ -1,5 +1,3 @@
-require 'semver'
```

Comment:
Do we know if any other modules on the forge `require 'semver'`? Ideally code written for 4.x will continue to work in 5.0. I wonder if providing an empty `semver.rb` would be useful (as in this case where it was required but never used)?